ibu-imager/seedcreator/seedcreator.go

```diff
@@ -212,50 +212,39 @@ func (s *SeedCreator) createContainerList() error {
 	s.log.Info("Saving list of running containers and catalogsources.")
 	containersListFileName := s.backupDir + "/containers.list"
 
-	// Check if the file /var/tmp/container_list.done does not exist
-	if _, err := os.Stat(common.BackupChecksDir + "/container_list.done"); os.IsNotExist(err) {
-		// Execute 'crictl images -o json' command, parse the JSON output and extract image references using 'jq'
-		s.log.Info("Save list of running containers")
-		args := []string{"images", "-o", "json", "|", "jq", "-r",
-			"'.images[] | if .repoTags | length > 0 then .repoTags[] else .repoDigests[] end'",
-			">", containersListFileName}
+	// Execute 'crictl images -o json' command, parse the JSON output and extract image references using 'jq'
+	s.log.Info("Save list of running containers")
```

Comment:
nit: list of images, since we're not just grabbing running containers

Reply:
Updated, thanks :)